bookstore/bookstore_config.py

```diff
@@ -39,7 +39,7 @@ class BookstoreSettings(LoggingConfigurable):
         config=True
     )
     published_prefix = Unicode("published", help="Prefix for published notebooks").tag(config=True)
-    enable_cloning = Bool(True, help="Enable cloning.").tag(config=True)
+    enable_s3_cloning = Bool(True, help="Enable cloning.").tag(config=True)
```

Comment:
Let's loop back and add these config options to the docstring.

Reply:
👍